src/python/pants/backend/native/subsystems/native_build_settings.py

```diff
@@ -0,0 +1,28 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.backend.native.subsystems.utils.mirrored_target_option_mixin import \
+  MirroredTargetOptionMixin
+from pants.subsystem.subsystem import Subsystem
+
+
+class NativeBuildSettings(Subsystem, MirroredTargetOptionMixin):
+  """Any settings relevant to a compiler and/or linker invocation."""
+  options_scope = 'native-build-settings'
+
+  mirrored_option_to_kwarg_map = {
+    'strict_deps': 'strict_deps',
+  }
+
+  @classmethod
+  def register_options(cls, register):
+    super(NativeBuildSettings, cls).register_options(register)
+
+    register('--strict-deps', type=bool, default=True, fingerprint=True, advanced=True,
+             help='The default for the "strict_deps" argument for targets of this language.')
```

Comment:
This help message does not really explain anything to the user who doesn't know about strict deps. Is the full explanation included anywhere in a help message?

Reply:
This is a trash message that I distinctly remember telling myself to fix a long time ago. Will fix.

Reply:
Should be fixed!